extensions/wikia/PaidAssetDrop/js/spec/paidAssetDrop.spec.js

```diff
@@ -90,8 +91,29 @@ describe('ext.wikia.paidAssetDrop.paidAssetDrop', function () {
 	it('URL param', function () {
 		var config = false;
 		spyOn(mocks, 'querystring').and.returnValue({
-			getVal: function () {
-				return '1';
+			getVal: function (param) {
+				return param === 'forcepad' ? '1' : undefined;
+			}
+		});
+		expect(getModule().isNowValid(config)).toEqual(true);
+	});
+
+	it('Disaster recovery', function () {
+		var config = false;
```

Comment:
What do you think about passing here a valid configuration? My idea is that if somebody instant globals condition this test will fail with valid config.